roxie/udplib/udptrs.cpp

```diff
@@ -671,6 +687,9 @@ class CSendManager : implements ISendManager, public CInterface
         {
             if (udpTraceLevel > 0)
                 DBGLOG("UdpSender: send_resend_flow started");
+#ifdef __linux__
+            setLinuxThreadPriority(2);
```

Comment:
This is really a separate change (and it's one where the effect is not easy to predict). Perhaps it should be in a separate PR and/or controlled by a separate flag.